meson.build

```diff
@@ -1,21 +1,64 @@
 project('hack-sound-server', version: '0.0.0')
 
+python = find_program('python3')
+pkgdatadir = join_paths(get_option('datadir'), meson.project_name())
+
 session_bus_services_dir = get_option('session-bus-services-dir')
 
 if session_bus_services_dir == ''
    session_bus_services_dir = join_paths(get_option('datadir'), 'dbus-1', 'services')
 endif
 
 install_data(
-    'com.endlessm.HackSoundServer.service',
+    'service/com.endlessm.HackSoundServer.service',
     install_dir: session_bus_services_dir
 )
 
+conf = configuration_data()
+conf.set('bindir', join_paths(get_option('prefix'), get_option('bindir')))
+conf.set('DATADIR', join_paths(get_option('prefix'), get_option('datadir')))
+conf.set('LIBDIR', join_paths(get_option('prefix'), get_option('libdir')))
+conf.set('PACKAGE', 'hack-sound-server')
+
+install_dir = join_paths(get_option('libdir'), 'hack-sound-server', 'python',
```

Comment:
use `meson.project_name()` for the first `hack-sound-server` (and consider factoring it out into a `pkglibdir` variable.)